testkit-backend/src/cypher-native-binders.js

```diff
@@ -58,7 +59,7 @@ export function cypherToNative (c) {
     case 'CypherString':
       return value
     case 'CypherInt':
-      return int(value)
+      return BigInt(value)
```

Comment:
There's no conditional here, which would imply that BigInt always is used.
I'm not very familiar with this part of the code so I might be mistaken.

Reply:
About the JSON serialization, I could add some notes on the documentation to users be aware of this when enable BigInt.

Reply:
Ah, great, that's what I figured.
Yes, I think a note on serialization in the docs would be good. Thanks!